examples/index.html

```diff
@@ -61,6 +61,9 @@
       href="../node_modules/at.js/dist/css/jquery.atwho.min.css"
       rel="stylesheet"
     />
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Cinzel:900" rel="stylesheet">
+	  <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
```

Comment:
Indentation again :)

Reply:
Made the changes, can we integrate ESLint in this?

Reply:
ESLint is not for HTML. It's for JavaScript. You could use Prettier.